### DISCUSSION TITLE
Michael Tiemann is now part of the dev team

### DISCUSSION THREAD

**Discussion by hgrecco** [hgrecco/pint#1860] [Announcements]
Please welcome @MichaelTiemannOSC as part of the pint and pint-pandas dev team.

Thanks for your awesome contributions!

**Comment by MichaelTiemannOSC** [comment]
Thank you all so much!

I have been a big fan of Pint ever since I first learned about it, and thrilled to discover Pint-Pandas as well.  In my day job (which is 100% volunteer work) I build tools for the analysis of climate-aligned financial data.  Most finance people only worry about two dimensions: time and money.  But ESG reports bring a menagerie of dimensions into the mix (tonnes of CO2e, barrels of oil, changes in temperature, etc), and Pint has been a godsend.

This is my second foray into the world of programming.  The first was in the 1980s, when I downloaded pre-release versions of the GNU C compiler and then proceeded to port that compiler to many chip architectures (including ns32k, m88k, SPARC, i960, i386, PA-RISC, MIPS, among others), as well as creating the GNU C++ compiler, and a few other things.

I retired from writing code about 5 years after starting the world's first open source software company (Cygnus).  Red Hat acquired Cygnus in 2000, I became their CTO, and worked at Red Hat until retiring last year.  In my retirement, I've taught myself Python, learned the ways of Pandas, Pint, and Pint-Pandas, and here I am.

Let's have some fun!
